apps/daimo-web/src/app/preview/route.tsx

```diff
@@ -0,0 +1,188 @@
+
+  return new ImageResponse(
+    (
+      <div
+        style={{
+          backgroundColor: "white",
+          height: "100%",
+          width: "100%",
+          display: "flex",
+          justifyContent: "center",
+          flexDirection: "column",
+        }}
+      >
+        <div
+          style={{
+            backgroundImage: `url("${daimoDomainAddress}/assets/preview-pixel-bg.svg")`,
+            height: "100%",
+            width: "100%",
+            display: "flex",
+            justifyContent: "space-between",
+            flexDirection: "column",
+          }}
+        >
+          <Content name={name} action={action} dollars={dollars} />
+          <Footer />
+        </div>
+      </div>
+    ),
+    {
+      width: 1200,
+      height: 630,
+    }
+  );
+}
+
+function Content({
+  name,
+  action,
+  dollars,
+}: {
+  name: string;
+  action: string | undefined;
+  dollars: string | undefined;
+}) {
+  return (
+    <div
+      style={{
+        height: "80%",
+        width: "100%",
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "space-between",
+      }}
+    >
+      <div
+        style={{
+          display: "flex",
+          width: "50%",
+          alignItems: "flex-end",
+          justifyContent: "center",
+          flexDirection: "column",
+          paddingRight: "24px",
+        }}
+      >
+        <UserBubble name={name} />
+      </div>
+      <div
+        style={{
+          display: "flex",
+          width: "50%",
+          alignItems: "flex-start",
+          justifyContent: "flex-start",
+          flexDirection: "column",
+          paddingLeft: "24px",
+        }}
+      >
+        <div
+          style={{
+            display: "flex",
+            alignItems: "flex-start",
+            justifyContent: "flex-start",
+            flexDirection: "column",
+            fontSize: 48,
+            color: "#717171",
+          }}
+        >
+          <div>{name}</div>
+          {action && <div>{action}</div>}
+        </div>
+        {dollars && (
+          <div
+            style={{
+              fontSize: 72,
+              color: "#262626",
+            }}
+          >
+            {`$${dollars}`}
+          </div>
+        )}
+      </div>
+    </div>
+  );
+}
+
+function Footer() {
+  return (
+    <div
+      style={{
+        height: "10%",
+        width: "100%",
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+        justifyItems: "center",
+        marginBottom: "48px",
+      }}
+    >
+      <img
+        alt="Daimo"
+        src={`${daimoDomainAddress}/logo-scalable.svg`}
+        height="100%"
+        style={{ marginRight: "24px" }}
+      />
+      <div
+        style={{
+          fontSize: 48,
+          color: "#262626",
+        }}
+      >
+        Daimo
+      </div>
+    </div>
+  );
+}
+
+// TODO: support rich profile custom images
+function UserBubble({ name }: { name: string }) {
```

Comment:
reusing `ContactBubble`  doesn't work?

Reply:
nope, I'm not even sure if it's because react native is not good at web or if doesn't support it. But given that `@vercel/og` doesn't even support tailwind, I think it's better either way to keep this code independent.